### PR TITLE
➕ Section 3: Operators

### PR DESCRIPTION
**Pull request by Palwisha-18**

### CODE REVIEW

File: 03_Operators/2_Remainder_Operator/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<h3>JavaScript Remainder Operator:</h3>
+
+- Remainder Operator (`%`)
+    * gives remainder of a number divided by another number.
+    * the sign of the remainder is the same as the sign of the dividend.
+    * if a value is not a number, the JavaScript engine will:
+        1. convert the value to a number using the Number() function.
+        2. apply the remainder function.
+
+- Modulos
```
Comment (ahmadjanan) [inline 03_Operators/2_Remainder_Operator/README.md]: Modulos -> Modulo

File: 03_Operators/2_Remainder_Operator/remainder_operator.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+let remainder = 10 % Infinity;
+console.log(remainder); // 10
+
+let remainder = 0 % 10;
+console.log(remainder); // 0
+
+let remainder = '10' % 3;
+console.log(remainder); // 1
+
+// difference between remainder and modulos
```
Comment (ahmadjanan) [inline 03_Operators/2_Remainder_Operator/remainder_operator.js]: `modulos` -> `modulo`

File: 03_Operators/2_Remainder_Operator/remainder_operator.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+const mod = (dividend, divisor) => ((dividend % divisor) + divisor) % divisor;
+
+// dividen and divisor have the same sign
```
Comment (ahmadjanan) [inline 03_Operators/2_Remainder_Operator/remainder_operator.js]: `dividen` -> `dividend`

File: 03_Operators/2_Remainder_Operator/remainder_operator.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+console.log('remainder:', 5 % 3); // 2
+console.log('modulo:', mod(5, 3)); // 2
+
+// dividen and divisor have the different signs
```
Comment (ahmadjanan) [inline 03_Operators/2_Remainder_Operator/remainder_operator.js]: `dividen` -> `dividend`

File: 03_Operators/5_Comparison_Operators/README.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+Comparison operators are used to compare two values and they return boolean values.
+
+- less than (`<`)
+- greater than (`>`)
+- less than or equal to (`<=`)
+- greater than or equal to (`>=`)
+- equal to (`==`)
+- not equal to (`!=`)
+- strict equal (`===`)
```
Comment (ahmadjanan) [inline 03_Operators/5_Comparison_Operators/README.md:11]: We should make it clear/mention that `==` compares only the value and not the data-type.
Whereas, `===` compares both value and data-type.

File: 03_Operators/7_Logical_Assignment_Operators/README.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<h3>JavaScript Logical Assignment Operators:</h3>
+
+- logical OR assignment (`||=`):
+    * assigns the right operand to the left operand if the left operand is falsy.
+- logical AND assignment (`&&=`)
+    * assigns the right operand to the left operand if the left operand is truthy.
+- Nullish coalescing assignment (`&&=`)
```
Comment (ahmadjanan) [inline 03_Operators/7_Logical_Assignment_Operators/README.md]: The Nullish coalescing operator is `??`. So,
`&&=` -> `??=`

File: 03_Operators/8_Nullish_Coalescing_Operator/nullish_coalescing_operator.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+// nullish coalescing operator
+
+const name = null ?? 'John';
+console.log(name); // 'John'
+
+const age = undefined ?? 28;
+console.log(age);
```
Comment (ahmadjanan) [inline 03_Operators/8_Nullish_Coalescing_Operator/nullish_coalescing_operator.js]: Let's add the console result to this line in a comment.

**Review by ahmadjanan — CHANGES_REQUESTED** [review]

File: 03_Operators/2_Remainder_Operator/README.md (hunk 1 above)
Comment (Palwisha-18) [reply 03_Operators/2_Remainder_Operator/README.md]: fixed.

File: 03_Operators/2_Remainder_Operator/remainder_operator.js (hunk 2 above)
Comment (Palwisha-18) [reply 03_Operators/2_Remainder_Operator/remainder_operator.js]: fixed.

File: 03_Operators/2_Remainder_Operator/remainder_operator.js (hunk 3 above)
Comment (Palwisha-18) [reply 03_Operators/2_Remainder_Operator/remainder_operator.js]: fixed.

File: 03_Operators/2_Remainder_Operator/remainder_operator.js (hunk 4 above)
Comment (Palwisha-18) [reply 03_Operators/2_Remainder_Operator/remainder_operator.js]: fixed.

File: 03_Operators/5_Comparison_Operators/README.md (hunk 5 above)
Comment (Palwisha-18) [reply 03_Operators/5_Comparison_Operators/README.md:11]: added

File: 03_Operators/7_Logical_Assignment_Operators/README.md (hunk 6 above)
Comment (Palwisha-18) [reply 03_Operators/7_Logical_Assignment_Operators/README.md]: fixed.

File: 03_Operators/8_Nullish_Coalescing_Operator/nullish_coalescing_operator.js (hunk 7 above)
Comment (Palwisha-18) [reply 03_Operators/8_Nullish_Coalescing_Operator/nullish_coalescing_operator.js]: added

**Review by ahmadjanan — APPROVED** [review]
LGTM 👍

**Pull request merged**